hbase-it/src/test/java/org/apache/hadoop/hbase/HBaseClusterManager.java

```diff
@@ -216,7 +217,7 @@ protected String findPidCommand(ServiceType service) {
     }
 
     public String signalCommand(ServiceType service, String signal) {
-      return String.format("%s | xargs kill -s %s", findPidCommand(service), signal);
+      return String.format("%s | xargs sudo kill -s %s", findPidCommand(service), signal);
```

Comment:
nice change, this help as long as the user is not the root user, and normally we have the hbase user when using Chaomonkey.